build.gradle

```diff
@@ -1,4 +1,12 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
+buildscript {
+    ext.kotlin_version = '<kotlin-version>'
```

Comment:
I think this is not being used anywhere, also there should be a proper version there?